src/vaults/vaults.service.ts

```diff
@@ -144,7 +144,31 @@ export class VaultsService {
     vault.apy = vault.sourcesApy.map((s) => s.apr).reduce((total, apr) => (total += apr), 0);
     vault.protocol = vaultDefinition.protocol ?? Protocol.Badger;
     vault.lastHarvest = pendingHarvest.lastHarvestedAt;
-    vault.yieldProjection = this.getVaultYieldProjection(vault, pendingHarvest);
+
+    const harvestProjection = this.getVaultYieldProjection(vault, pendingHarvest);
+    const passiveSources = sourcesApr.filter((s) => s.type === SourceType.TradeFee || s.type === SourceType.Emission);
```

Comment:
once the projections based on the harvest values are calculated, 

we can then grab the sources which are passive and considered to always be active to enrich the data returned by harvest / yield estimates

this is intended to help the projection values properly return the estimates they should be giving back 

NOTE: in the case of the badger rewards, this is the pro rata APR 

so while its useful for some general idea of what is going on the UI for example should not utilize this metric and instead favor the sources objects as a source of truth to derive an individual user's level of compensation wrt emitted badger rewards